pulsar-testclient/src/main/java/org/apache/pulsar/proxy/socket/client/PerformanceClient.java

```diff
@@ -61,172 +62,161 @@
 import picocli.CommandLine;
 import picocli.CommandLine.Command;
 import picocli.CommandLine.Option;
-import picocli.CommandLine.ParameterException;
 import picocli.CommandLine.Parameters;
 
-public class PerformanceClient {
+@Command(description = "Test pulsar websocket producer performance.")
+public class PerformanceClient extends CmdBase {
 
     private static final LongAdder messagesSent = new LongAdder();
     private static final LongAdder bytesSent = new LongAdder();
     private static final LongAdder totalMessagesSent = new LongAdder();
     private static final LongAdder totalBytesSent = new LongAdder();
     private static IMessageFormatter messageFormatter = null;
-    private CommandLine commander;
 
-    @Command(description = "Test pulsar websocket producer performance.")
-    static class Arguments {
+    @Option(names = { "-h", "--help" }, description = "Help message", help = true)
+    boolean help;
```

Comment:
Unnecessary.

Reply:
The `mixinStandardHelpOptions = true` has been set in the root command.

Reply:
Other commands also have this issue, please check.

Reply:
Done.